### ISSUE TITLE
Repo name is not consistent

### ISSUE THREAD

**Issue by chadmetcalf** [openfootball/leagues#14]
I apologize if the repo name has been brought up before. I looked through the issues & PRs and couldn't find any discussion. I imagine that 
```openfootball/major-league-soccer```
was chosen because MLS spans two countries. Is there a less surprising way to deal with the problem? 

We could rename this repo ```openfootball/us-united-states``` & prop up a README at ```openfootball/ca-canada``` explaining the situation. Or, maintain an updated fork at ```openfootball/ca-canada```. Or if we really feel clever we could lean on github's [repo redirect](https://github.com/blog/1508-repository-redirects-are-here) feature to forward ```openfootball/ca-canada``` to ```openfootball/us-united-states```.

This would also provide a place to begin to collect US Open and Canadian Championship and lower division data that is consistent with the rest of ```openfootball``` repo naming convention.

**Comment by geraldb** [comment]
Good point. Yes, the reason for major-league-socces is / was that it includes the united states and canada. I'd say changing the name will not get you more contributions. If the name keeps you from contributing - by all means rename it. Cheers.
